pulpcore/cli/common/generic.py

```diff
@@ -390,10 +390,15 @@ def label_command(**kwargs: Any) -> click.Command:
     if "name" not in kwargs:
         kwargs["name"] = "label"
     decorators = kwargs.pop("decorators", [name_option, href_option])
+    needs = kwargs.pop("needs", [("core", "3.10.0")])
 
     @click.group(**kwargs)
-    def label_group() -> None:
-        pass
+    @pass_pulp_context
+    def label_group(pulp_ctx: PulpContext) -> None:
+        for plugin in needs:
+            min_ver = plugin[1] if len(plugin) >= 2 else None
+            max_ver = plugin[2] if len(plugin) == 3 else None
+            pulp_ctx.needs_plugin(plugin[0], min_ver, max_ver)
```

Comment:
We can do something fancy with a namedtuple here. But maybe not today...

Reply:
Ask and you shall receive.